### PR TITLE
Django 1.9 support

### PR DESCRIPTION
**Pull request by mvantellingen**

### CODE REVIEW

**Comment by mvantellingen** [comment]
django-treebeard still need some fixes for Django 1.9

File: .travis.yml [hunk 1]
```diff
@@ -14,9 +14,10 @@ cache:
 language: python
 
 python:
-    - '2.7'
-    - '3.3'
-    - '3.4'
+    - 2.7
+    - 3.3
+    - 3.4
+    - 3.5
```
Comment (maikhoepfel) [inline .travis.yml]: <3

**Comment by mvantellingen** [comment]
All tests should now pass on Django 1.9 when using https://github.com/g3rd/django-treebeard

**Comment by alexkahn** [comment]
What did you all do with treebeard? Did you just write it into the app or does that fork have a PyPI entry?

**Comment by itbabu** [comment]
@mvantellingen Just a small note:
haystack doesn't support Django 1.9 yet.
E.g. I get an error every time results are found.

https://github.com/django-haystack/django-haystack/search?q=1.9&type=Issues&utf8=%E2%9C%93

**Pull request merged**